app/homepage/ui.test.js

```diff
@@ -8,64 +9,50 @@ fixture('Show Home Page');
 
 test('should render the homepage hero', async (t) => {
   pageSetup(t);
-
   const homepageSection = Selector('[data-test-id="homepage-hero"] > section');
   const title = homepageSection.find('h1');
   const description = homepageSection.find('p');
 
   await t
     .expect(homepageSection.count).eql(1)
     .expect(title.innerText).eql('Buying Catalogue')
-    .expect(description.innerText)
-    .eql('The GP IT Futures procurement framework has replaced GP System of Choice (GPSoC). Use our Buying Catalogue to find out what high-quality IT systems and services are available.');
+    .expect(description.innerText).eql('The GP IT Futures procurement framework has replaced GP System of Choice (GPSoC). Use our Buying Catalogue to find out what high-quality IT systems and services are available.');
 });
 
 test('should render the about us section', async (t) => {
   pageSetup(t);
-
   const aboutUsSection = Selector('[data-test-id="about-us"]');
   await t
     .expect(aboutUsSection.find('h3').innerText).eql('About us')
     .expect(aboutUsSection.find('p').innerText).eql('The NHS Digital Buying Catalogue is an online marketplace platform. It can be used to find out which clinical IT systems are available to buy from the GP IT Futures framework.')
-    .expect(aboutUsSection.find('p').nth(1).innerText)
-    .eql('The GP IT Futures procurement framework, which replaces the GP System of Choice (GPSoC) framework, will make a wider range of systems available. The systems have been assessed against a set of Capabilities and Standards to help improve safety, quality and efficiency in GP surgeries.')
-    .expect(aboutUsSection.find('p').nth(2).innerText)
-    .eql('The functionality of the Buying Catalogue will be enhanced and the number of available products – known as Solutions - will increase over time.');
+    .expect(aboutUsSection.find('p').nth(1).innerText).eql('The GP IT Futures procurement framework, which replaces the GP System of Choice (GPSoC) framework, will make a wider range of systems available. The systems have been assessed against a set of Capabilities and Standards to help improve safety, quality and efficiency in GP surgeries.')
+    .expect(aboutUsSection.find('p').nth(2).innerText).eql('The functionality of the Buying Catalogue will be enhanced and the number of available products – known as Solutions - will increase over time.');
```

Comment:
I actually prefer it the way it was before where we can split it over multiple lines

Reply:
I disagree as it is inconsistent and less confusing if the expect and equal statement are on the same line. It is just annoying that we have really long strings in some of the tests which makes the line really long. How about slit over 2 lines but the second line indented so it looks like it is at least associated with the above line?

Reply:
yes the indents would be better 👍

Reply:
will leave it as is at the moment as it will be fixed with the string extraction 👍